packages/component/src/Styles/StyleSet/RenderMarkdown.ts

```diff
@@ -79,6 +82,10 @@ export default function createMarkdownStyle() {
         flexDirection: 'column'
       },
 
+      '& :is([data-math-type=block], [data-math-type=inline]) > span': {
```

Comment:
I'd look into removing the span

Reply:
Should this be something handled on your side of work right now? Say, `mathChunk` should not render any tags.

Reply:
On the extension side we do render only root plus what's KaTeX returned. Looks like we need to adjust the renderer by opting to `renderToDomTree` instead of `renderToString`:

```
var renderToString = function renderToString(expression, options) {
  var markup = renderToDomTree(expression, options).toMarkup();
  return markup;
};
```

I can look into it.

Reply:
Not quite... :rofl:

![image](https://github.com/user-attachments/assets/c56ddbcb-650f-486a-a971-9e504d35ed0a)